docs/modules/ROOT/pages/explanations/backup.adoc

```diff
@@ -0,0 +1,62 @@
+= K8up Backup Methods
+
+K8up currently implements three ways to do backups:
+
+. xref:how-tos/backup.adoc[_PVC backups_], which work by mounting the underlying `PV` of each `PVC` and reading its files.
+. xref:how-tos/application-aware-backups.adoc[_Backup commands_], which print the content, that should be backed up, to STDOUT.
+. The special xref:how-tos/prebackuppod.adoc[`PreBackupPod`], which is run as part of a backup.
+
+This explanation guide briefly explains each backup method.
+
+== PVC Backup
+
+This is the most straight-forward backup approach.
+What k8up basically does is that it looks at all the `PVC` definitions in a namespace.
+For every `PVC` it creates a `Job` in that same namespace.
+That `Job` will launch a `Pod`, which mounts the respective `PV`.
+All the content of that `PV` is then backed-up.
+
+This method's advantage is that it is dead-easy to use and works for a surprising large amount of use-cases.
+It does not work in some cases though.
+More precise, it does not work when files are kept open for a long period of time, like databases do.
+It does also not work for content that is not stored in the cluster, like a managed database that is offered by a service provider.
+
+== Application-Aware Backups
+
+The backup command is defined an annotation on your `Pod`.
+When k8up does the backup, it will start that command in the context of your `Pod`.
+It uses the same method as if you are running `kubectl exec POD -- COMMAND ARGS`.
+K8up will then collect everything that is written to `STDOUT`.
+The collected content is then stored in the configured backup storage as text file.
```

Comment:
```suggestion
The collected content is then stored in the configured backup storage as file.
```

Why "text file"? If the output to STDOUT is binary, it will also be stored in a binary format...